usaspending_api/database_scripts/job_archive/management/commands/dev_4109_repair_awards.py

```diff
@@ -0,0 +1,122 @@
+import logging
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from usaspending_api.common.helpers.timing_helpers import Timer
+
+logger = logging.getLogger("console")
+
+CHECK_MISSING_AWARDS = """
+    SELECT tf.{field}
+    FROM {table} AS tf
+    LEFT OUTER JOIN awards a on a.generated_unique_award_id = tf.unique_award_key
+    WHERE a.id IS NULL;
+"""
+
```

Comment:
No comment explaining where it comes from and when it's obsolete?

Reply:
hectic day